storybook/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import {CSSTransition} from 'react-transition-group';
+import "../css/Button.css";
+import classNames from "classnames";
+
+export class Button extends Component {
```

Comment:
So PureComponent was actually the correct one to extend, I just wanted to have you learn more about it